doc/source/whatsnew/v0.24.0.txt

```diff
@@ -8,15 +8,20 @@ v0.24.0 (Month XX, 2018)
 New features
 ~~~~~~~~~~~~
 
-- ``ExcelWriter`` now accepts ``mode`` as a keyword argument, enabling append to existing workbooks when using the ``openpyxl`` engine (:issue:`3441`)
+.. _whatsnew_0240.enhancements.excel_writer_mode:
```

Comment:
just leave it like it was, its not necessary to make a sub-section, just a single entry is fine.